### ISSUE TITLE
Release ARM Binaries

### ISSUE THREAD

**Issue by c-nv-s** [pglet/pglet#23]
Please could you also provide arm7 and arm64 binaries as part of your releases

**Comment by FeodorFitsner** [comment]
Sure, will add that. Are you looking into using Pglet on M1 or Raspberry PI?

**Comment by c-nv-s** [comment]
There are a few SBCs and even a couple of arm VMs which I would be interested in using Pglet ( which I keep pronouncing piglet haha ) for, which are from Sinovoip and FriendlyElec.
Come to think of it, I suppose in order to satisfy the Raspberry crowd you might as well just release arm6 instead of 7.

**Comment by FeodorFitsner** [comment]
Got it, thanks for your feedback! We are going to make a new release tomorrow with ARM7/ARM64 (as well as M1) binaries and a lot of new controls.

Will look into Bulma. In the meantime, I'm going to blog about why we chose Fluent UI.

**Comment by FeodorFitsner** [comment]
While I'm working on the release (notes, docs, tests) you can download `arm64` binary from "Artifacts" of build results: https://ci.appveyor.com/project/pglet/pglet/build/job/ery3ck1uvdan7val/artifacts

If you unpack `pglet` executable in `pglet-linux-arm64.tar.gz` to any location from `$PATH` it will be picked up by existing `pglet.sh` script. Let me know how that worked.

Btw, if you can check what do you get on ARMv7 machine when running `uname -m`? I'm trying to do a switch for various architectures in Bash installer script. Thanks!

**Comment by c-nv-s** [comment]
1. running uname -m 
on arm64 bare metal : aarch64
on arm7 bare metal : armv7l
on arm7 container launched from arm64 machine : armv8l
on arm64 container launched from arm64 machine : aarch64

2. Unable to confirm whether pglet.sh automatically picks up the untarred pglet-linux-arm64.tar.gz file because of the following error on arm64:

```
$ curl -O https://pglet.io/pglet.sh
$ source pglet.sh
$ Error: Unsupported architecture aarch64. Only x64 binaries are available.
```

**Comment by c-nv-s** [comment]
also, if I remember correctly, some arm7 architectures may also return a machine code of armv7hf for the hard floating point variant.

**Comment by FeodorFitsner** [comment]
Thank you, I've updated the installation script accordingly!

Pglet v0.2.0 has been just released and I'm still working on release notes - there is a lot of new stuff like, for example, this [Grid Monstrocity](https://repl.it/@pglet/grid-example) :scream:

You can simply use an updated script which supports ARM now:

    curl -O https://pglet.io/pglet.sh

Let me know how it worked!

**Comment by FeodorFitsner** [comment]
Just wanted to share this example in Bash I've been working on: https://repl.it/@pglet/bash-proc-mon
It uses Grid control from Pglet v0.2.0.

**Comment by c-nv-s** [comment]
> Just wanted to share this example in Bash I've been working on: https://repl.it/@pglet/bash-proc-mon
> It uses Grid control from Pglet v0.2.0.

thanks all examples are going to be very useful.
this is a really awesome project and I think it could open up a tonne of possibilities.
please let me know the blog post link as well when it's complete :-)
I'll test the arm installs later!

**Comment by c-nv-s** [comment]
> Let me know how it worked!

I can confirm both arm7/64 install, binaries and examples are working.
Feel free to let me know if you need anymore arm related testing.

**Comment by FeodorFitsner** [comment]
Thanks, you helped a lot!

I've just published Pglet 0.2.0 release notes with Bash demos for each new control: https://pglet.io/blog/pglet-0-2-0#new-controls

I'm going to close this issue as, eventually, ARM got supported, but stay around :)
